config/routes.rb

```diff
@@ -37,7 +37,7 @@ def matches?(_)
           get 'eligibility' => 'listings#eligibility'
         end
       end
-      get 'trk' => 'email#confirmation'
+      get 'trk' => 'listing_interest#index'
```

Comment:
I actually like confirmation better - I think it's clearer

Reply:
Ok. I was thinking it might be confused with the email confirmations we have today for when a user signs up for an account.

Reply:
listing_interest#confirmation?